test/services/EnglishUIBuilderSpec.scala

```diff
@@ -999,7 +945,7 @@ class UIBuilderSpec extends BaseSpec with ProcessJson {
   }
 
   trait InputTest {
-
+    implicit val lang: Lang = Lang("en")
```

Comment:
Is this not defined in the English language trait?

Reply:
removed, there were many more, must of missed this one